example/example_test.go

```diff
@@ -169,7 +169,7 @@ func TestE2E(t *testing.T) {
 	t.Logf("%#v\n", docs)
 
 	i := dc.ChangeFeed(nil)
-	docs, err = i.Next(ctx)
+	docs, err = i.Next(ctx, 1)
```

Comment:
I discovered this hadn't been updated, so I got it working again.

Reply:
thanks!